src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -124,19 +124,25 @@ public override ValueTask DisposeAsync()
 
     /// <inheritdoc/>
     public override async Task TerminateInstanceAsync(
-        string instanceId, object? output = null, CancellationToken cancellation = default)
+        string instanceId, TerminateInstanceOptions? options = null, CancellationToken cancellation = default)
     {
+        if (options is null)
+        {
+            options = new TerminateInstanceOptions();
```

Comment:
I'd prefer we avoid allocating unnecessary objects like this. Instead you can either:

1. Have a `static` empty instance of `TerminateInstanceOptions`
2. OR split into defaults in this method:

``` CSharp
string? output = options?.Output;
bool recursive = options?.Recursive ?? false;

// rest of function
```

Reply:
Done